app/models/manageiq/providers/autosde/storage_manager/event_catcher/runner.rb

```diff
@@ -34,7 +34,7 @@ def queue_event(event)
   def event_to_hash(event, ems_id)
     {
       :event_type               => event.event_type,
-      :source                   => "AUTOSDE",
+      :source                   => PhysicalStorage.find_by(:ems_ref=>event.storage_system).name,
```

Comment:
`:source` must be the provider type for the rest of the event switchboard to work so this will break all automate event handlers.

Also the event catcher should not perform any queries like this for a few reasons but two primary among them being performance and the fact that events might be for inventory which is not in the database yet.  This is filled in by the event_handler worker already (https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_event.rb#L140-L142, https://github.com/ManageIQ/manageiq/blob/0a7431d0b1b52cca9b05839c055bc25b51dd9506/app/models/ems_event.rb#L106)

Also if the physical storage fails to be found here (e.g. if it hasn't been refreshed yet) then this `.find_by` will return `nil` and `nil.name` will throw an exception.

Reply:
@agrare I saw that there is `physical_storage_name`  in the event stream:
![image](https://user-images.githubusercontent.com/6840118/188305955-cee4b867-bdcb-4e06-b262-737fb3bf90e8.png)
Can you give me direction how to display it in the monitor's UI, something like that: ![image](https://user-images.githubusercontent.com/6840118/185965130-9e4fe75f-4d7f-41ed-8d96-7ce6a4c29486.png)